storage/src/mmr/mem.rs

```diff
@@ -177,6 +178,60 @@ impl<H: CHasher> Mmr<H> {
         Ok(self.size())
     }
 
+    /// Change the digest of an existing leaf. NOTE that this will change the root hash and
```

Comment:
nit: I've usually written something like this as:

```rust
/// Change the digest of an existing leaf.
///
/// # Warning
///
/// This will change the root hash and...
```

Reply:
done